### ISSUE TITLE
"No image available" for many products

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/openfoodfacts-hungergames-react#1]
Example below:
![image](https://user-images.githubusercontent.com/562352/53697836-dad17800-3dd5-11e9-8d9f-a3289e111ebe.png)

**Comment by CharlesNepote** [comment]
Nearly no pictures for the USA.

**Comment by ArnaudBarre** [comment]
Yes I know for USA, @teolemon can probably explain why. Did this happen often for other countries ?

**Comment by SantosSi** [comment]
Yes, it happens for Germany too.

**Comment by teolemon** [comment]
USA, perfectly normal, we imported the USDA without pictures. 
For other countries, no ideas.

**Comment by CharlesNepote** [comment]
Isn't it possible to filter products without a picture (ie exclude them)? Or at least to have an option to do so?
